src/Orchard.Web/Modules/Orchard.OpenId/Startup.cs

```diff
@@ -27,13 +28,13 @@ public class Startup : StartupBase
 
         public Startup(
             ShellSettings shellSettings,
-            IAppDataFolder appDataFolder,
+            IHostingEnvironment environment,
```

Comment:
Okay, looks like `ShellsRootContainerName` and `ShellsContainerName` were introduced yesterday just after I "rebased" the `oauth2` branch. I'll fix that, thanks!